### ISSUE TITLE
Option to be Notified of Every Toot from Specified Accounts

### ISSUE THREAD

**Issue by jkuester** [tootsuite/mastodon#9735]
I wanted to see what folks thought about adding the ability to receive a notification whenever one of a specified set of accounts posts a new status.  

Sometimes when you are following a bunch of accounts, your Home feed fills up. However there can be particular accounts that you want to make sure you do not miss their posts. It would be nice to receive a notification when a new status is posted for those accounts.  

Basically my proposal would be to add the functionality to generate notifications about posts from a sub-set of the list of accounts you are following.  

As far as the UI goes for populating that sub-set I am open to suggestions. The straightforward approach would just be to add the list in the Settings > Notifications menu with a search bar. You could search though the accounts you follow to add them to the list.

**Comment by Cassolotl** [comment]
I think this might be a duplicate of *Feature Request: notifications for all toots of particular users #4890*?

**Comment by jkuester** [comment]
Whoops, you are right!  

Duplicate of #4890
